pages/studentAssessmentInstanceHomework/studentAssessmentInstanceHomework.js

```diff
@@ -80,6 +92,15 @@ router.post('/', function(req, res, next) {
             if (ERR(err, next)) return;
             res.redirect(req.originalUrl);
         });
+    } else if (req.body.__action == 'quitGroup') {
+        var params = {
+            assessment_instance_id: res.locals.assessment_instance.id,
+            user_id: res.locals.user.user_id,
+        };
+        sqldb.query(sql.quit_group, params, function(err, result) {
```

Comment:
to get rid of the linter error that `result` is unused:

```suggestion
        sqldb.query(sql.quit_group, params, function(err, _result) {
```